README.md

```diff
@@ -134,6 +135,70 @@ import "@asyncapi/react-component/styles/default.css";
 import "@asyncapi/react-component/styles/default.min.css";
 ```
 
+### Custom styles
+The AsyncAPI React component reset a number of styles and does not set any global fonts. This component provides the option to define your custom `font-family` and other styling.
```

Comment:
@thim81 The component doesn't reset `a number of styles` 😅

Reply:
I ll take it out 🤦‍♂️